utils/auth.py

```
@@ -0,0 +1,69 @@
+
+    }
+    return jwt.encode(payload, SECRET_KEY, algorithm=ALGORITHM)
+
+
+def decode_token(token: str) -> str:
+    try:
+        payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
+        user_id = payload.get("sub")
+        if not user_id:
+            raise HTTPException(status_code=401, detail="Invalid token")
+        return user_id # 토큰 발급 주체 반환
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)